htdocs/contrat/list.php

```diff
@@ -108,7 +109,8 @@
 $staticcontratligne = new ContratLigne($db);
 
 if ($search_status == '') {
-	$search_status = 1;
+	// if nothing selected, we select all
+	$search_status = array('1', '2', '3', '4');
```

Comment:
If no filter is selected, we must not include a test or for all values but you must just have no criteria in the sql at all